app/controllers/landing_page_controller.rb

```diff
@@ -183,12 +191,8 @@ def parse_int(int_str_or_nil)
     nil
   end
 
-  def community_id(request)
-    request.env[:current_marketplace]&.id
-  end
-
-  def community_default_locale(request)
-    request.env[:current_marketplace]&.default_locale
+  def current_community(request)
+    @current_community ||= request.env[:current_marketplace]
```

Comment:
I guess this change is because layouts/google_analytics_script relies on global `@current_community`? That's a style we haven't been using for a while because it's just a really really bad idea for code structuring. How far off would we be from reworking  the partial to take the community as a local? Or even better take the data it needs from the community object as a hash?